docs/api-reference/next.config.js/headers.md

```diff
@@ -50,7 +50,7 @@ module.exports = {
 `headers` is an async function that expects an array to be returned holding objects with `source` and `headers` properties:
 
 - `source` is the incoming request path pattern.
-- `headers` is an array of header objects with the `key` and `value` properties.
+- `headers` is an array of headers to add to the response - each header is an object with `key` and `value` properties.
```

Comment:
```suggestion
- `headers` is an array of response header objects with the `key` and `value` properties.
```

I'm not sure I like the "`headers` is an array of headers" as an explanation of the term.

Reply:
Fixed.